rust/headless-client/src/linux.rs

```diff
@@ -10,31 +10,62 @@ use connlib_shared::{
 use firezone_cli_utils::setup_global_subscriber;
 use futures::{SinkExt, StreamExt};
 use secrecy::SecretString;
-use std::{
-    future,
-    net::IpAddr,
-    path::{Path, PathBuf},
-    str::FromStr,
-    task::Poll,
-};
+use std::{future, net::IpAddr, path::PathBuf, str::FromStr, task::Poll};
 use tokio::{
     net::{UnixListener, UnixStream},
     signal::unix::SignalKind,
 };
 use tokio_util::codec::LengthDelimitedCodec;
 
+const SOCK_PATH: &str = "/run/firezone-client.sock";
```

Comment:
This is where Docker puts their socket, so I copied that. Apparently `/run` is the new version of `/var/run`

Reply:
Perhaps leave this as a doc-comment?